tests/test_api.rs

```diff
@@ -4845,15 +4845,27 @@ fn test_object_get_property_names() {
     proto_obj.set(scope, js_proto_test_str, js_null);
     obj.set_prototype(scope, proto_obj.into());
 
-    let own_props = obj.get_own_property_names(scope).unwrap();
+    let own_props = obj
+      .get_own_property_names(
+        scope,
+        v8::GetPropertyNamesArgsBuilder::new().build(),
+      )
+      .unwrap();
     assert_eq!(own_props.length(), 1);
     assert!(own_props.get_index(scope, 0).unwrap() == js_test_str);
 
-    let proto_props = proto_obj.get_own_property_names(scope).unwrap();
+    let proto_props = proto_obj
+      .get_own_property_names(
+        scope,
+        v8::GetPropertyNamesArgsBuilder::new().build(),
```

Comment:
Any chance we can make this API nicer by being able to pass `Default::default()` as the second argument? It's a lot of typing if we don't want to configure the filtering

Reply:
@bartlomieju didn't know that was a thing in rust, done, PTAL!